hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/BaseEntity.java

```diff
@@ -19,6 +20,10 @@
  */
 public interface BaseEntity extends Serializable, Identifiable<Long> {
 
+    static Long getIdOrNull(final BaseEntity entity) {
```

Comment:
Can we get rid of this method? This helper method, should not be part of this interface.